package.json

```diff
@@ -77,16 +79,16 @@
 		"@lezer/python": "^1.1.1",
 		"@lezer/rust": "^1.0.0",
 		"@lezer/xml": "^1.0.0",
-		"@monaco-editor/react": "^4.1.1",
+		"@monaco-editor/react": "4.1.1",
```

Comment:
This is really silly, but npm install this as 4.1.1, pnpm as 4.5.0, which leads to some bugs with the types. I removed the caret to make it more consistent.